.github/workflows/ci.yaml

```
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  merge_group:
+    branches: [ main ]
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  test_unit:
+    strategy:
+      matrix:
+        python-version: ["3.9","3.10","3.11"]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: make install-test
+    - name: Run unit tests
+      run: |
+        pip install -e .
```

Does this need to be installed in editable mode? 
```suggestion
        pip install  .
```

---

No. That was just my default for installing.